contracts/p1/Broker.sol

```diff
@@ -23,10 +22,10 @@ contract BrokerP1 is ComponentP1, IBroker {
     using SafeERC20Upgradeable for IERC20Upgradeable;
     using Clones for address;
 
-    uint48 public constant MAX_AUCTION_LENGTH = 604800; // {s} max valid duration - 1 week
+    uint48 public constant MAX_AUCTION_LENGTH = 60 * 60 * 24 * 7; // {s} max valid duration, 1 week
     /// @custom:oz-upgrades-unsafe-allow state-variable-immutable
     // solhint-disable-next-line var-name-mixedcase
-    uint48 public immutable MIN_AUCTION_LENGTH; // {s} 20 blocks, based on network
+    uint48 public constant MIN_AUCTION_LENGTH = 60; // {s} 60 seconds auction min duration
```

Comment:
It's actually different, but not in the same way. We used 20 blocks previously because we need 20 steps of precision for good numbers in auctions. Now that it's gone, we need a better default of what it means. The protocol now assumes working on a {s} timescale and for that reason I chose 60 seconds here for enough steps as a default.

That said, we'll almost never use a number that low, and easily by a factor of 10 at the very least. Down to discuss this, but honestly don't think it's worth it.

Reply:
best reason for upping it is that a low number like this is a potential foot-gun.  60 seconds is still just 5 blocks, which is 25% of what we had previously. i'd be in favor of something like 5 minutes? open to arguments against raising it too high though (only one i can see is "L2's r fast")

Reply:
I don't disagree with you, but we have a bunch of other places where minimum values are potentially a foot-gun. That being said, I'm down to update this, I don't think it's a big deal at all. Any number (over 20) we choose here is going to be ok. It's just a minimum, we'll _never_ see this in action.

It's 25% of the previous value on mainnet but actually 1.5x the value on Base. So not quite the same comparison. (I get your point though)

And yeah, "L2s are fast" is exactly the reason I chose 60s haha

Reply:
I think this is expected and we shouldn't expect to do better. The ranges are no longer a function of the chain, so whatever numbers we come up with are going to be too restrictive on one type of chain (fast L2s) and not restrictive enough on another type of chain (ETH L1). I think this means the ranges aren't there to guarantee good performance or a well-functioning RToken, they're there for something weaker.

Reply:
^ only downside to this refactor

Reply:
Yep, exactly my reasoning.